setup.py

```diff
@@ -68,8 +68,12 @@
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=[
-        'molecule==1.25.1',
-        'ansible==2.3.2',
+        'ansible==2.4',
         'docker',
+        'docker-py',
```

Comment:
Do you rememebr why this dependency is required? I briefly looked into the state of the opened molecule v2 PRs with Ansible 2.5/2.6 following the discussion in https://github.com/openmicroscopy/ansible-role-basedeps/pull/5#issuecomment-421947293.  However the initial `destroy` phase failed with https://github.com/metacloud/molecule/issues/1384. After removing `docker-py` from `setup.py` and re-installing this meta-package, as suggested in the issue and by the output of `molecule --debug test`, the tests passed.

Reply:
From what I remember it was a mess due to breaking changes between docker and docker-py, we definitely shouldn't have both. It'll need testing.

Reply:
I don't remember what else, but docker-py is certainly critical for docker-compose integration.

Reply:
Indeed `docker` seems to be a full rewrite of `docker-py` and both packages certainly incompatible changes.

Looking at the requirements of `docker-compose` and various issues includes https://github.com/metacloud/molecule/issues/1347, https://github.com/ansible/ansible/issues/22993: 

- `docker-compose` nows declares `docker` as a dependency instead of `docker-py` since 2.10.0
- the Ansible Docker module used to recommend `docker-py` as a dependency. Recent Ansible versions seem to have included full support for `docker` and https://github.com/ansible/ansible/pull/44856 is updating the official doc accordingly.

Switching to `docker` across the board sounds like the long-term option anyways. For this PR, I wonder whether the decision here is related to the targetted version of Ansible https://github.com/openmicroscopy/ansible-role-basedeps/pull/5#issuecomment-422849660. If we decide to go for Ansible 2.6, `docker` should work both for `compose` and the `ansible` Docker modules. If we stick to Ansible 2.4, it might be that `docker-py` is the best choice.

Reply:
The following combinations work on travis with idr/deployment https://github.com/IDR/deployment/pull/135
- ansible==2.5.9 docker-py==1.10.6 molecule==2.18.0
- ansible==2.6.4 docker-py==1.10.6 molecule==2.18.0
- ansible==2.6.4 docker==3.5.0 molecule==2.18.0